app/views/batch_processes/show_parent.html.erb

```diff
@@ -0,0 +1,76 @@
+<h1>Parent Batch Process Detail</h1>
+<p id="notice"><%= notice %></p>
+<p>
+  <strong>Batch Process ID</strong>
+  <%= link_to @batch_process.id, batch_process_path(@batch_process) %>
+</p>
+<p>
+  <strong>Parent Object</strong>
+  <%= if @parent_object.class == ParentObject
```

Comment:
how is parent object ever not a parent object? (this is not meant as a deep a philosophical question or riddle ;-) )